curator/Dockerfile

```diff
@@ -16,28 +16,38 @@ ENV HOME=/opt/app-root/src \
     CURATOR_TIMEOUT=300 \
     CURATOR_VER=5.8.1 \
     container=oci \
-    LC_ALL=en_US.UTF-8 
+    LC_ALL=en_US.UTF-8
 
-LABEL io.k8s.description="Curator elasticsearch container for elasticsearch deletion/archival" \
-  io.k8s.display-name="Curator ${CURATOR_VER}" \
-  io.openshift.tags="logging,elk,elasticsearch,curator"
+USER 0
+COPY . ${HOME}
 
-RUN yum install -y python3 && \
-    yum clean all && \
-    ln -sf /usr/bin/python3 /usr/bin/python
+#
+# TODO Remove after openshift/release provides ubi7/python36 base image
+#
+RUN ${HOME}/install-centos7-deps.sh
 
-COPY . ${HOME}
 RUN mkdir -p $(dirname "$CURATOR_CONF_LOCATION") && \
     touch ${CURATOR_CONF_LOCATION} && \
     chmod -R u+x ${HOME} && \
     chgrp -R 0 ${HOME} && \
     chmod -R g=u ${HOME}
+
 WORKDIR ${HOME}/vendor
-RUN pip3 install $(ls . | grep -v curator) -q --no-index --find-links . && \
-    pip3 install elasticsearch-curator* --no-index -q && \
-    rm -rf $HOME/vendor && \
-    ls /usr/local/bin/curator
+RUN pip install $(ls . | grep -v curator) -q --no-index --find-links . && \
+    pip install elasticsearch-curator* --no-index -q && \
+    rm -rf $HOME/vendor
 
 WORKDIR ${HOME}
 USER 1001
 CMD ["sh", "run.sh"]
+
+LABEL io.k8s.description="Curator elasticsearch container for elasticsearch deletion/archival" \
+    io.k8s.display-name="Curator ${CURATOR_VER}" \
+    io.openshift.tags="logging,efk,elasticsearch,curator" \
+    com.redhat.component="logging-curator${CURATOR_VER}-container" \
```

Comment:
@sosiouxme is it correct to say this label and all those after it can be removed?